docs/demo/demo.js

```diff
@@ -185,7 +185,7 @@ function setOptions(opts) {
   $optionsElem.value = JSON.stringify(
     opts,
     (key, value) => {
-      if (value && typeof value === 'object' && Object.getPrototypeOf(value) !== Object.prototype) {
```

Comment:
I think this is needed since `null` will throw an error in `Object.getPrototypeOf(null) !== Object.prototype`

You can change it to `value !== null &&` if you want

Reply:
ah, nice catch, done